resources/js/views/settings/Application.vue

```diff
@@ -0,0 +1,220 @@
+<template>
+  <div>
+    <b-container fluid>
+      <h3>
+        {{ $t('settings.application.title') }}
+        <b-button id="application-save-button"
+                  class="float-right"
+                  variant="success"
+                  @click="updateSettings(settings)"
+                  :disabled="isBusy">
+          <span><i class="fas fa-save mr-2"></i>{{ $t('settings.application.save') }}</span>
+        </b-button>
+      </h3>
+
+      <hr>
+
+      <!--Logo Settings-->
+      <b-form-group
+        label-for="application-logo-input"
+        :description="$t('settings.application.logo.description')"
+        :state='fieldState("logo")'
+      >
+
+        <template v-slot:label>
+          <span><i class="fas fa-camera-retro mr-3"></i></span>
+          {{ $t('settings.application.logo.title') }}
+        </template>
+
+        <b-row align-v="baseline" class="my-3">
+          <b-col sm="6" lg="3" class="text-center">
+            <b-img
+              :src="settings.logo"
+              class="my-2"
+              rounded="0"
+              alt="application-logo-preview"
+              width="150"
+              height="100"
+              fluid
+            >
+            </b-img>
+          </b-col>
+          <b-col sm="6" lg="9">
+            <b-input-group>
+              <b-form-input id="application-logo-input"
+                            :placeholder="$t('settings.application.logo.hint')"
+                            v-model="settings.logo"
+                            :disabled="isBusy"
+                            :state='fieldState("logo")'
+              >
+              </b-form-input>
+            </b-input-group>
+          </b-col>
+        </b-row>
+
+        <template slot='invalid-feedback'>
+          <div v-html="fieldError('logo')"></div>
+        </template>
+      </b-form-group>
+
+      <!--Room limit settings-->
+      <b-form-group
+        label-for="application-room-limit-input"
+        :description="$t('settings.application.roomLimit.description')"
+        :state='fieldState("room_limit")'
+      >
+
+        <template v-slot:label>
+          <span><i class="fas fa-person-booth mr-3"></i></span>
+          {{ $t('settings.application.roomLimit.title') }}
+        </template>
+
+        <b-form-input id="application-room-limit-input"
```

Comment:
Please also add a checkbox for unlimited rooms, like it was done for the group room limit